playbooks/inventory/group_vars/all.yml

```diff
@@ -118,6 +118,10 @@ wai_nginx_openresty_tag: 1.0.0-stable
 wai_cache_image: varnish
 wai_cache_image_tag: 6.5.1
 
+# WAI Kong Image
+wai_kong_image: webanalyticsitalia/wai-kong
```

Comment:
Qui usiamo l'immagine in https://github.com/AgID/wai-portal/tree/api-portal/containers/kong oppure mettiamo un Dockerfile in questo repo?
In dev c'è un [`kong-setup.sh`](https://github.com/AgID/wai-portal/blob/api-portal/containers/kong/kong-setup.sh) per fare un setup iniziale, ne possiamo prevedere uno analogo anche qui?

Reply:
Qui usiamo quella che su wai-portal ... devo portare qui la build del container ?

Reply:
@giafar qui all'ultimo meeting dicevamo di portare dentro la build con una soluzione simile a quella in https://github.com/AgID/wai-portal/tree/api-portal/containers/kong che abbia quindi:
- avvio delle migrations di kong;
- impostazione di plugin, servizi e routes parametrizzati